--- goggles-0.9.0-orig/build/configdetect	2006-04-02 09:11:29.000000000 -0500
+++ goggles-0.9.0/build/configdetect	2006-05-15 21:54:24.732875000 -0500
@@ -18,6 +18,8 @@
 	else
   	CONFIG_FILE="build/config.linux"
   fi
+elif [ ${OSNAME:0:6}="CYGWIN" ] && [ -f build/config.cygwin ] ; then
+	CONFIG_FILE="build/config.cygwin"
 elif [ $OSNAME="HP-UX" ] && [ -f build/config.hpux ] ; then
 	CONFIG_FILE="build/config.hpux"
 elif [ $OSNAME="SunOS" ] && [ -f build/config.solaris ] ; then
